PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
+    URL = "http://127.0.0.1:8080/account/authenticate"
+    #URL = "https://api.develop.socotra.com/account/v1/authenticateInternal
```

Comment:
If you are going to have *inline* quotes, there are a couple of rules:
```
    URL = 'https://foo.bar.io'  # Some fake url
```
1) There should be two spaces after the code before the `#` character
2) There should be one space between the `#` and the comment

The second rule would set off a linter here.